service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DiaryPlanStatementMapperTest.java

```diff
@@ -74,7 +74,8 @@ public void setUp() throws IOException {
         messageContext.initialize(bundle);
         messageContext.getIdMapper().getOrNew(ResourceType.Organization, new IdType("6D340A1B-BC15-4D4E-93CF-BBCB5B74DF73"));
 
-        diaryPlanStatementMapper = new DiaryPlanStatementMapper(messageContext, codeableConceptCdMapper);
+        diaryPlanStatementMapper =
+            new DiaryPlanStatementMapper(messageContext, codeableConceptCdMapper, new ParticipantMapper());
```

Comment:
Using actual `ParticipantMapper`, by analogy to `EhrExtractMapperTest`. Since it's a very simple mapper, mocking it wouldn't add much value, if any - we'd need to set it up to prepare some output. Can as well be the actual output.